frontend/css/layout.css

```diff
@@ -64,12 +77,97 @@ li a {
     padding-top: var(--spacing-m);
 }
 
-.ideas {
-    text-align: center;
+.search-icons {
+    flex-direction: column;
+    align-items: center;
+}
+
+.menu-button {
+    margin-right: var(--spacing-m);
+}
+
+.nav-icons {
+    margin-right: var(--spacing-l);
+}
+
+@media (min-width: 63.75rem) {
+    .menu {
+        justify-content: flex-start;
+    }
+
+    .container {
+        flex-wrap: nowrap;
+        display: flex;
+    }
+
+    .search-bar {
+        width: 50%;
+    }
+
+    .search-container {
+        display: flex;
+        order: 2;
+    }
+
+    .logo {
+        order: 1;
+    }
+
+    .nav-icons {
+        order: 3;
+    }
+}
+
+@media (min-width: 1180px) {
+    .menu-button {
+        display: none;
+    }
 }
 
 /* Categories section */
 
+.sidebar {
+    position: fixed;
+    list-style-type: none;
+    top: 0;
+    left: 0;
+    height: 100vh;
+    width: 250px;
+    z-index: 1;
+    display: none;
+    backdrop-filter: blur(10px);
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+    background-color: rgba(255, 255, 255, 0.2);
+}
+
+.sidebar li {
+    width: 100%;
+}
+
+.sidebar a {
+    width: 100%;
+}
+
+.close {
+    height: 2rem;
+    width: 2rem;
+}
+
+@media (max-width: 1180px) {
+    .hideOnMobile {
```

Comment:
I was thinking about this class. If we keep it, can we use the kebab-case naming convention as for the other classes?

An alternative which I would prefer personally is to not have a generic "hide on mobile" class, but to make a case by case decision for what should be shown/hidden in certain breakpoints.

Reply:
Thanks Sara - i've removed that class now since we can target the whole list using the child combinator (categories-list > li)